django_ipam/static/django-ipam/js/subnet.js

```diff
@@ -1,10 +1,17 @@
 /*jslint browser:true */
 /*globals onUpdate*/
 
-function dismissAddAnotherPopup(win) {
+function normalizeIP(ip_address) {
+    'use strict';
+    return ip_address ? ip_address.split(":").join("").split(".").join("") : null;
```

Comment:
please use single quotes and be consistent with the quoting style used in the codebase.

Reply:
okay @nemesisdesign, let me fix this